update custom.sample.php

xslugx opened the pull request:
Add extra features and requests.
Added coin price API for coin values
added tables and bootstrap 4 styling with Navbar for further development.

xslugx committed:
update custom.sample.php

Add extra features and requests.
Added coin price API for coin values
added tables and bootstrap 4 styling with Navbar for further development.

Elompenta commented:
@xslugx seems good, thank for the update - i test the code and merge it to the master trunk

Elompenta commented on custom.sample.php:
```diff
@@ -15,16 +16,206 @@
+$link='https://api.coinmarketcap.com/v1/ticker/'.$coin.'?convert='.$price;
+$data=file_get_contents($link);
+$json=json_decode($data);
+$usd=$json[0]->price_usd;
```
wring variable naming - hardcoded to "USD" - because its an API an in Line 29/30 you tell the User he can Switch the Currenc y;)

xslugx replied:
Ah it should be $currency?

Elompenta replied:
this was my first opinion too..
then i have think das we use "currency" in this Project for the digital-currency *lol*

$real_money
$real_currency
? at the Moment i have no clue for a correct naming :_P

xslugx replied:
How about $fiatCurrency

Elompenta replied:
what is the meaning of that? ;)
In the first Moment i think of a car from italy (Sorry!) ;)

Elompenta commented on custom.sample.php:
```diff
@@ -15,16 +16,206 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
```
Please use local files without cdn networks

Elompenta commented on custom.sample.php:
```diff
@@ -15,16 +16,206 @@
-	</hr>
-	<?php print_r($ant->get('paymentHistory')); ?>
-</pre>
+<html lang="en">
```
your look & feel is nice! i like it 👍

Elompenta commented on custom.sample.php:
```diff
@@ -15,16 +16,206 @@
+    <tr>
+      <td><?php print "$balance LTC"; ?></td>
+      <td><?php print "$daily LTC"; ?></td>
+      <td><?php print $daily*7; print "LTC"; ?></td>
```
Hardcoded Currency? ;)

Elompenta commented on custom.sample.php:
```diff
@@ -15,16 +16,206 @@
+      <td><?php print $daily*30; print "LTC"; ?></td>
+    </tr>
+    <tr>
+      <td><?php print "$";print round($balanceUSD,2); ?></td>
```
Hardcoded Currency

Elompenta commented on custom.sample.php:
```diff
@@ -15,16 +16,206 @@
+    <th colspan="4"><center>Last 10 Payments</center></th>
+    </tr>
+    <tr>
+      <th scope="col">Time & Date</th>
```
& Need to be escaped

xslugx replied:
It stands for "and" in English

Elompenta replied:
Yeah - i think this is the meaning in the whole world (Chinese not i think so :_D)
"&" is an Special escape charactere in html.

If you will print a "&" you should use the string `&amp;`
https://dev.w3.org/html5/html-author/charref

Elompenta commented on custom.sample.php:
```diff
@@ -15,16 +16,206 @@
+  </tbody>
+</table>
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
```
please also use local libaries - feel free to add a new global css and js folder

Elompenta reviewed (changes requested):
Very nice look & feel - but some changes are required to realized all user scenarios

xslugx commented:
Looks great! This was some of the stuff I needed to fix but wasn't sure how. Thanks again!

Elompenta commented:
i will work on this too - thanks for help

xslugx commented:
I can edit this later tonight. I forgot to edit the function previously as well.

xslugx committed:
Update custom.sample.php

Fix hard coded use, still need to add variable for currency symbols and crypto names so crypto is no longer hard coded

xslugx committed:
Update custom.sample.php

soft code fiat symbols and crypto abbreviations for easier usage

xslugx committed:
update custom.sample.php

Add local css and js files, add responsive layout for better mobile compatibility. striped rows for readability

Elompenta commented:
ive created a new branch from your pull request.
https://github.com/Elompenta/antpool-php-api/tree/xslugx-patch-2

feel free to check it, befor we will it to merge to master branch

Elompenta commented:
closed - please check my post: https://github.com/Elompenta/antpool-php-api/commit/7ae015d0d2c5bfe71b6b0eb224073d5dd5175a22
